algebras/algebra/src/main/scala/endpoints/algebra/Methods.scala

```diff
@@ -14,4 +14,6 @@ trait Methods {
 
   def Patch: Method
 
+  def Options: Method
```

Comment:
I dont follow. I added the `options` http method because sooner or later someone will expect to have it. It has nothing to do with removing the algebra but its better to have it than dont, isnt it?

Reply:
Sorry, my comment was a bit short :)

The first part was: is it needed in order to remove the documented algebra? But I guess not :)

The second part was a thought: I think our server interpreters could provide `options` requests handling based on the descriptions of the endpoints. The idea would be that users wouldn’t have to explicitly describe endpoints with the `options` method, but servers would be able to reply to such requests in a useful way.